pkg/image/generated/clientset/typed/image/v1/fake/fake_imagestreammapping.go

```diff
@@ -2,6 +2,7 @@ package fake
 
 import (
 	v1 "github.com/openshift/origin/pkg/image/apis/image/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Why are we using meta_v1 in some places and metav1 in others for the same package? Seems confusing

Reply:
this is generated code